examples/js/exporters/GLTFExporter.js

```diff
@@ -837,7 +837,7 @@ THREE.GLTFExporter.prototype = {
 		 * @param  {THREE.Material} material Material to process
 		 * @return {Integer}      Index of the processed material in the "materials" array
 		 */
-		function processMaterial( material ) {
+		function processMaterial( material, aoMapUVIndex ) {
```

Comment:
I think calling this parameter `uvCount` or `numUVs` would be (1) more general, and (2) more representative of how it's computed. Its values could be `1 | 2` in that case.